tests/xtimer_remove/main.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       timer test application
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "msg.h"
+#include "thread.h"
+#include "xtimer.h"
+
+#define N 3
+
+int main(void)
+{
+    puts("xtimer_usleep_until test application.\n");
+
+    kernel_pid_t me = thread_getpid();
+
+    for (int n = 0; n < N; n++) {
+        printf("Setting %u timers, removing timer %u/%u\n", N, n, N);
```

Comment:
tests/xtimer_remove/main.c:36: warning (invalidPrintfArgType_uint): %u in format string (no. 2) requires an unsigned integer given in the argument list.